### ISSUE TITLE
uniqueID is null. Not creating ColorLight object

### ISSUE THREAD

**Issue by rohitmanusis** [SteveyO/Hue-Emulator#19]
uniqueID is null. Not creating ColorLight object

**Comment by SteveyO** [comment]
uniqueid was added to Bridge Firmware 1.4.   The emulator mostly supports up to bridge firmware 1.3.  I have updated the Emulator Home Page to further emphasize this.

"Not creating ColorLight object" means nothing to me.

**Comment by roshanraj** [comment]
Hi SteveyO, have added uniqueid to a fork version of yours. See If that goes well merge it with your code (https://github.com/roshanraj/Hue-Emulator)

**Comment by SteveyO** [comment]
Hi Rishanraj,

I have just committed the code for the Unique Id.  Feel free to pull it and let me know if it works for you.
I didn't run your code but at a glance, there were some issues.   It looked like you were generating a Random Unique Id for every get.  I fixed this by fixing the UniqueId in the default (config-3bulbs.json) JSON config and in the addNewLights method (in Controller.java)  I then use your random Mac function (with a couple of minor modifications).

I will do a v0.6 emulator build sometime this week, maybe later.

Steve
